tests/recovery/recoverySecurity.test.ts

```
@@ -0,0 +1,375 @@
+import { getUserRecoveryAuditHistory } from '../../src/services/recovery/auditService';
+
+const prisma = new PrismaClient();
+
+describe('Recovery Security E2E Tests', () => {
+  let testUser: any;
+  let testDeviceFingerprint: DeviceFingerprint;
+  let testDeviceFingerprint2: DeviceFingerprint;
+
+  beforeAll(async () => {
+    // Create test user with passcode
+    const passcodeHash = await bcrypt.hash('test1234', 10);
```

ESLint

[error] Replace `'test1234'` with `"test1234"`

(prettier/prettier)